ansible/roles/microshift-start/tasks/main.yml

```diff
@@ -18,7 +23,9 @@
   - name: cleanup microshift data
     become: yes
     ansible.builtin.shell: echo 1 | {{ microshift_cleanup_bin }} --all
-  when: cleanup_microshift | default('true') | bool
+  when:
+    - cleanup_microshift | default('true') | bool
+    - microshift_active.stdout | int != 0
 
 - name: disable microshift service
```

Comment:
Error reported on this step: `fatal: [microshift-dev]: FAILED! => {"changed": false, "msg": "Could not find the requested service microshift: host"}`
You can check the whole output in Jenkins: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/microshift/job/microshift-performance-tests/71/console

Reply:
Rerun it again, skipping the task `disable microshift service`, but the execution is stuck in the task:
`TASK [microshift-start : run the microshift boot script] ***********************`

I ssh into the VM and run:
```
[microshift@microshift-dev ~]$ sudo systemctl status microshift.service
Unit microshift.service could not be found.
```
Jenkins execution log: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/microshift/job/microshift-performance-tests/72/consoleFull

Reply:
This means that it didn't install MicroShift. In the logs it shows you've skipped the whole role involved in this PR.

Reply:
And also the `manage_repos` role, as that's what installs the pre-release repo (or any of them).
For now just set it, and then we can change it in a different PR if that's what we determine.

Reply:
It seems the mirror repo for pre-releases is failing. Can you take a look, please? 

```
TASK [manage-repos : install EPEL repo] ****************************************
10-11 16:37:46.696  fatal: [microshift-dev]: FAILED! => {"changed": false, "msg": "Failed to download metadata for repo 'microshift-4.14-for-rhel-9-mirrorbeta-x86_64-rpms': Cannot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried", "rc": 1, "results": []}
```

https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/microshift/job/microshift-performance-tests/75/console

Reply:
The logs show you set `microshift_version=4.14.0-rc.3` which is not a version that exists in the repo. There is `4.14.0-rc.2` or  `4.14.0-rc.4`